pkg/scheduler/factory.go

```diff
@@ -651,3 +579,19 @@ func (b *binder) Bind(binding *v1.Binding) error {
 	klog.V(3).Infof("Attempting to bind %v to %v", binding.Name, binding.Target.Name)
 	return b.Client.CoreV1().Pods(binding.Namespace).Bind(binding)
 }
+
+// GetPodDisruptionBudgetLister returns pdb lister from the given informer factory. Returns nil if PodDisruptionBudget feature is disabled.
+func GetPodDisruptionBudgetLister(informerFactory informers.SharedInformerFactory) (pdbLister policylisters.PodDisruptionBudgetLister) {
```

Comment:
prefer non-named return values.
And does it need to be exported?

Reply:
this one does not, the other one does, so thought I would be consistent.